andela/mrm-mobile
# #160379519 Device should keep awake while Converge is running

#### What does this PR do?
This PR delivers functionality that keeps the device from sleeping while the converge application is active.

#### Description of Task to be completed?
Initially, the device would go to sleep after a given period of time of inactivity regardless of the converge application running.

This PR delivers the ability for the application not to sleep as long as the converge application is running.

#### How should this be manually tested?
1. Clone the repository onto your machine and build the app. 
2. Make sure that the tablet is connected and configured to your machine correctly.
3. In the device settings, select Display.
4. Select Sleep and choose 30s to have the application sleep after 30s.This will ensure that the device sleeps after the given period incase it has been inactive.
5. Run the converge application and get it installed on the tablet.
6. Leave the application inactive for more than 30s. You will notice that the device never sleeps.
7. Run any other application and leave it inactive for more than 30s, the device will go to sleep after 30s.

#### Background context
This functionality cannot be tested on an emulator since the emulator is always active.

[Delivers #160379519]
